src/components/Breadcrumb.vue

```diff
@@ -0,0 +1,107 @@
+      required: true,
+      validator: val => val.type && val.id,
+    },
+    loading: {
+      type: Boolean,
+      default: false,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    append: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  inject: ['girderRest'],
+  asyncComputed: {
+    breadcrumb: {
+      default: { root: {}, path: [] },
+      async get() {
+        this.$emit('update:loading', true);
+        const breadcrumb = { root: {}, path: [] };
+        const { id, type, name } = this.location;
+        if (type === 'folder') {
+          // The last breadcrumb isn't returned by rootpath.
+          if (name) {
+            breadcrumb.path.unshift(this.location);
+          } else {
+            const { data } = await this.girderRest.get(`folder/${id}`);
+            breadcrumb.path.unshift(this.extractCrumbData(data));
+          }
+          // Get the rest of the path.
+          const { data } = await this.girderRest.get(`folder/${id}/rootpath`);
+          data.reverse().forEach((crumb) => {
+            const { object } = crumb;
+            const entity = this.extractCrumbData(object);
+            if (entity.type === 'folder') {
+              breadcrumb.path.unshift(entity);
+            } else {
+              breadcrumb.root = entity;
+            }
+          });
+        } else {
+          const { data } = await this.girderRest.get(`${type}/${id}`);
+          breadcrumb.root = this.extractCrumbData(data);
+        }
+        this.$emit('update:loading', false);
+        return breadcrumb;
+      },
+    },
+  },
+  methods: {
+    extractCrumbData(object) {
+      return {
+        name: object._modelType !== 'user' ? object.name : object.login,
+        id: object._id,
+        type: object._modelType,
+        parentId: object.parentId,
+        parentType: object.parentCollection,
+      };
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+v-breadcrumbs.girder-breadcrumb-component.pl-0.pt-0.pb-0
+  v-icon.mdi-24px(
+      :class="{ disabled: disabled }",
+      slot="divider",
+      color="accent") {{ $vuetify.icons.chevron }}
+  v-breadcrumbs-item(
+      :class="{ disabled: disabled }",
+      @click.native="$emit('changelocation', breadcrumb.root)")
+    v-icon.mdi-24px(color="accent") {{ $vuetify.icons.globe }}
+    | &nbsp; {{ breadcrumb.root.name }}
+  v-breadcrumbs-item(
+      :class="{ disabled: disabled }",
+      v-for="item in breadcrumb.path",
+      :key="`${item.id}.crumb`",
+      @click.native="$emit('changelocation', item)") {{ item.name }}
+  v-breadcrumbs-item(
+      v-for="item in append",
+      :key="`${item.id}.crumb`") {{ item }}
+</template>
+
+<style lang="scss">
+.girder-breadcrumb-component {
+  font-weight: 700;
+
+  &.v-breadcrumbs li:nth-child(2n) {
+    padding: 0 4px !important;
+  }
+
+  li.disabled a,
+  li.v-breadcrumbs__divider i.disabled {
+    color: gray !important;
+    cursor: default !important;
+  }
+}
```

Comment:
For this I'm going to tag in @jtomeck 

Those styles don't affect the FileBrowser, they're for the breadcrumb in the Create Folder popup where the breadcrumb is static.  That PR is #48 

In general, it seems useful to be able to "disable" the breadcrumbs so they don't appear interactive.

```
v-breadcrumbs.headline
```

This overrides the typography set in App.vue, and would prevent consumers of this library from setting global font themes.  It doesn't look like Vuetify supports any font other than Roboto.

The font themes @jtomeck wants to use is "Barlow Condensed"

Reply:
> This overrides the typography set in App.vue, and would prevent consumers of this library from setting global font themes. It doesn't look like Vuetify supports any font other than Roboto.

The proper way to change Vuetify's font, I believe, is to update stylus variables in a custom `main.styl` described [here](https://vuetifyjs.com/en/style/theme#stylus-guide).

There is the additional problem that headings (like headline) are dealt with separately. I have been able to make this [main.styl](https://gist.github.com/jeffbaumes/5ea3ae818a8cae4bcb64579ef2487d44) change the font for all things, including headings. Redefining `$headings` is not ideal (copy/paste from Vuetify stylus) but I could not seem to get it work without it.

It also requires stylus webpack rules, but it seems Vuetify is [moving to SCSS](https://github.com/vuetifyjs/vuetify/issues/5558) in the future, so it would be a temporary thing to support both in this example app bundle.

Reply:
> This overrides the typography set in App.vue

I don't understand what you mean, isn't the `headline` class part of Vuetify's typography API?

Reply:
What happens is that just setting the font on the body does not override all of Vuetify's fonts. There is a more specific CSS for `headline` in Vuetify that is Roboto and "overrides" the customized Barlow Condensed body CSS font in App.vue (which in turn was overriding Vuetify's body font of Roboto). My comment addresses this by explaining a way to truly change Vuetify's default font, so that Roboto no longer appears in the CSS, both for body text and all headings.

Reply:
Ah, thanks for the explanation. I assumed "headline" and the related typography classes mostly controlled size, weight, color, rather than typeface.

@jtomeck how important is it for us to override Vuetify's prescribed typeface? If we want to do it, I'd like to make sure we do it the way @jeffbaumes describes as well so that we can use Vuetify's typography classes rather than having to use manual, ad hoc CSS rules for those things, in the interest of using a consistent theme that is easy to control centrally.

Reply:
I'm personally not a huge fan of "Barlow Condensed".  I find myself squinting at it.  

We could certainly scale it up, but that would (i think) involve changing roughly 25 font sizes from [variables](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/stylus/settings/_variables.styl) that have already been optimized for Roboto.

Reply:
I'll point out that the font can be changed entirely through Vuetify Stylus/CSS customization. Our library of components should not change the font internally; apps should be able to decide what to do with Vuetify. There are several choices to make: 1. What font do we want our example app showcase to have? 2. What font do we want the new Girder UI to have? 3+. For project X, what font does it want to have? The answers to these should be able to be different for each use case, though the answers to 1 and 2 will probably set precedent for downstream projects.

Reply:
My two cents here. The Barlow Condensed feels a little bit too condensed for me if used everywhere, meanwhile default Roboto do use a lot of space in actual use. So, maybe we could try the Roboto Condensed as suggested by @jtomeck? So that when Project X decides to use default Roboto, it won't be too big a change.